maestro-orchestra/src/test/resources/YamlCommandReaderTest/023_labels.yaml

```diff
@@ -112,6 +112,18 @@ appId: com.example.app
 - waitForAnimationToEnd:
     timeout: 4000
     label: "Wait for the thing to stop spinning"
+- swipe:
+    direction: DOWN
+    label: "Swipe down a bit"
+- addMedia:
+      files:
+          - "023_image.png"
+      label: "Add a picture to the device"
+- setAirplaneMode:
+      value: enabled
+      label: "Turn on airplane mode for testing"
+- toggleAirplaneMode:
+      label: "Toggle airplane mode for testing"
```

Comment:
These are all of the labelled commands that would otherwise have been outliers.
I've been through and checked that all of the other commands are covered by this test.